tests/main/views/test_buyers.py

```diff
@@ -1871,6 +1871,56 @@ def test_show_awarded_brief_summary_page_for_live_and_expired_framework(self, da
             assert not document.xpath('//a[contains(text(), "Delete")]')
 
     @pytest.mark.parametrize('framework_status', ['live', 'expired'])
+    def test_cancel_link_present_on_closed_brief_summary(self, data_api_client, framework_status):
+        self.login_as_buyer()
+        data_api_client.get_framework.return_value = api_stubs.framework(
+            slug='digital-outcomes-and-specialists',
+            status=framework_status,
+            lots=[
+                api_stubs.lot(slug='digital-specialists', allows_brief=True),
+            ]
+        )
+        data_api_client.get_brief.return_value = api_stubs.brief(status="closed")
+
+        res = self.client.get(
+            "/buyers/frameworks/digital-outcomes-and-specialists/requirements/digital-specialists/1234"
+        )
+
+        page_html = res.get_data(as_text=True)
+        document = html.fromstring(page_html)
+
+        assert res.status_code == 200
+        assert "Cancel requirement" in document.text_content()
```

Comment:
Rather than having these separate two tests, could we add in 1-line assertions to the first 5 tests in `TestBriefSummaryPage`?

Reply:
I think those tests do too much already.
If `test_show_awarded_brief_summary_page_for_live_and_expired_framework` fails I'd have no idea what was wrong.
If `test_cancel_link_present_on_closed_brief_summary` I'd know exactly what had happened.